orders/models.py

```diff
@@ -0,0 +1,22 @@
+from tkinter import CASCADE
+from django.db       import models
+
+from core.models     import TimeStampModel
+from products.models import Product
+from users.models    import User
+
+class Order(TimeStampModel):
+    user         = models.ForeignKey(User, on_delete=models.CASCADE)
+    order_status = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = 'orders'
+
+class Order_Product(models.Model):
```

Comment:
```suggestion
class OrderProduct(models.Model):
```
Class 이름에는 언더바(_)를 사용하지 않습니다!